TestApp/set-ndk-version.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Set the NDK version in the android section of the build.gradle file.
+
+PROPERTY_FILE="$ANDROID_HOME/ndk-bundle/source.properties"
```

Comment:
Let's use "$ANDROID_NDK_HOME" instead of "$ANDROID_HOME/ndk-bundle". You will need to set the variable as part of your `install-ndk-version.sh` script.

Reply:
done